.storybook/custom-addons/strictmode/index.js

```diff
@@ -0,0 +1,37 @@
+import {addons, makeDecorator} from '@storybook/addons';
+import {getQueryParams} from '@storybook/client-api';
+import React, {StrictMode, useEffect, useState} from 'react';
+
+function StrictModeDecorator(props) {
+  let {children} = props;
+  let [isStrict, setStrict] = useState(getQueryParams()?.strict === 'true' || false);
+
+  useEffect(() => {
+    let channel = addons.getChannel();
+    let updateStrict = (val) => {
+      setStrict(val);
+    };
+    channel.on('strict/updated', updateStrict);
```

Comment:
I would've like to used the storybook globals but for some reason it stopped working